pkg/models/system_intake.go

```diff
@@ -100,6 +101,18 @@ const (
 	SystemIntakeStepDECISION SystemIntakeStep = "DECISION_AND_NEXT_STEPS"
 )
 
+// Humanize replaces underscores with spaces and converts an uppercased word to a capitalized one
+func (step SystemIntakeStep) Humanize() string {
+	upperStepSlice := strings.Split(string(step), "_")
+	var wordSlice []string
+	for _, word := range upperStepSlice {
+		word = strings.ToLower(word)
+		upperWord := strings.ToUpper(string(word[0])) + word[1:]
```

Comment:
This can cause a panic if `step` is an empty string, but it's probably not worth worrying about, given that this method _should_ only have our defined constants passed in.

Reply:
addressed in [82d5c96](https://github.com/CMSgov/easi-app/pull/2225/commits/82d5c96f57bdc0214b90f8a5e6a605e52b05be2e)